java/src/plume/WeakHasherMap.java

```diff
@@ -117,7 +118,9 @@
  * @see		java.lang.ref.WeakReference
  */
 
-@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking
+@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking.
+// Annotated for lock checking as an experiment in annotating non-typesafe code.
```

Comment:
I see. Yes, approximately half of all the annotations written are in these two files with non-typesafe code. I wanted to get a sense of how many @SuppressWarnings would be needed in this scenario (only a handful it turns out), though this did not uncover any bugs in the Lock Checker.